### ISSUE TITLE
Unable to import graphql definitions with graphql-import syntax

### ISSUE THREAD

**Issue by artemzakharov** [dotansimha/graphql-code-generator#1156]
**Describe the bug**
I'm unable to use `graphql-import` syntax to import definitions into my schema file anymore. 

Syntax I'm referring to (this is at the top of my schema file):

`# import CartWhereUniqueInput from "prisma.graphql"`

Result of running gql-gen:

![image](https://user-images.githubusercontent.com/25446299/51077370-25026c80-1663-11e9-88c2-e2a5be639a31.png)

I'm filing this as a bug rather than a feature because I've been using this syntax for a while and have only recently run into an issue.

**To Reproduce**
1. Import a graphql definition using the above syntax
2. Attempt to run the generator

**Expected behavior**
I expect the definition to be imported and resolved successfully. 

**Environment:**
 - OS: Windows
 - Codegen: 0.15.2
 - Node: 10.9.0

**Additional context**
I'm thinking that `graphql-import` was simply removed from this library because Prisma uses the same syntax, and that still works perfectly fine for me, while gql-gen has started having issues.

**Comment by kevin-lindsay-1** [comment]
Some issue here; if I generate a `.d.ts` file and attempt to import those types, `gql-gen` fails, but when is simply generate a `.ts` file it works just fine.

**Comment by kamilkisiela** [comment]
I followed your steps and I couldn't reproduce the issue https://github.com/dotansimha/graphql-code-generator/commit/bae9f87eeb09b84ca18ee1af9d5159ad8ae5a569

**Comment by kamilkisiela** [comment]
@artemzakharov could you prepare a small reproduction?

**Comment by artemzakharov** [comment]
I found the issue - it turns out that comments I had at the top of my schema file (above the import statements) were somehow interfering with the import! I tried removing everything other than the import statements, and it suddenly started working. Seems like import statements have to be above any and all comments you have, or they won't work. Guess this is what happens when we don't have proper import support 😂
